packages/storybook/src/ui/organisms/table/table.test.tsx

```diff
@@ -29,4 +29,29 @@ describe('Table', () => {
   it('Renders Horizontally', () => {
     render(<Table definition={summaryDefinition} records={summaryRecordsList} orientation="HORIZONTAL" />)
   })
+
+  it('Renders horizontal table with multiple row headers. total 13 columns to be visible', () => {
+    render(<Table definition={mockSummaryDefinition} records={mockSummaryRecordsList} orientation="HORIZONTAL" />)
+
+    expect(screen.getAllByRole('columnheader').length).toBe(13)
+  })
+
+  it('Renders horizontal table with two data rows', () => {
+    render(<Table definition={mockSummaryDefinition} records={mockSummaryRecordsList} orientation="HORIZONTAL" />)
+
+    expect(screen.getAllByRole('rowgroup')[0].children.length).toBe(2)
+  })
+
+  it('Renders Vertically', () => {
+    render(<Table definition={mockSummaryDefinition} records={mockSummaryRecordsList} />)
+
+    expect(screen.getByRole('table')).toBeVisible()
+  })
+
+  it('Renders Vertical table with 5 rows in tbody', () => {
+    render(<Table definition={mockSummaryDefinition} records={mockSummaryRecordsList} />)
+
+    expect(screen.getByRole('table').children[1].tagName).toBe('TBODY')
+    expect(screen.getByRole('table').children[1].children.length).toBe(5)
+  })
```

Comment:
Good stuff so far - can we add some tests for the subheaders on Horizontal tables? 
![image](https://user-images.githubusercontent.com/63285990/195859306-14fac8aa-bc28-4f7c-921d-f0455bfb8667.png)

Reply:
cool